coderd/database/queries/siteconfig.sql

```diff
@@ -70,3 +70,10 @@ SELECT value FROM site_configs WHERE key = 'oauth_signing_key';
 -- name: UpsertOAuthSigningKey :exec
 INSERT INTO site_configs (key, value) VALUES ('oauth_signing_key', $1)
 ON CONFLICT (key) DO UPDATE set value = $1 WHERE site_configs.key = 'oauth_signing_key';
+
+-- name: GetDismissedHealthchecks :one
+SELECT value FROM site_configs WHERE key = 'dismissed_healthchecks';
```

Comment:
Should we coalesce this to `[]`?

Reply:
I thought about it. There is one drawback of this approach: with `[]` we will close the structure for extra properties since it will accept only array items. I'm happy to change it to `[]` if you think that it is fair.

Reply:
I mean, we could also coalesce it to `{}` and then start off with `{"dismissed": [...]}` if you want to keep it open. My main point here was to avoid downstream consumers having to deal with a potential `nil`.

Reply:
Yes, this will be open. Should I rename it to `health_settings` instead?

Reply:
👍 Sounds good, I could imagine other persistent health-related site configs going in there.

Reply:
Alright, converted it into `health_settings` with default value `{}`.